src/classes/CON_ContactMergeTDTM_TEST2.cls

```diff
@@ -290,4 +312,113 @@ public with sharing class CON_ContactMergeTDTM_TEST2 {
             system.assertEquals(2, accWinner.npo02__NumberOfClosedOpps__c);            
         }
     }
+
+    /*******************************************************************************************************
+    * @description Verifies rollup is calculted on merge
+    */
+    @isTest
+    private static void contactMergeShouldRecalculteRollup() {
+        List<Contact> contactsToMerge = buildContactsWithOppsForAccount();
+
+        Test.startTest();
+        merge contactsToMerge[0] contactsToMerge[1];
+        Test.stopTest();
+
+        List<Contact> contacts = [SELECT npo02__NumberOfClosedOpps__c FROM Contact];
+        System.assertEquals(1, contacts.size(), 'There should only be one contact after merge');
+        System.assertEquals(2, contacts[0].npo02__NumberOfClosedOpps__c, 'The rollup should have been calculated');
+    }
+
+    /*******************************************************************************************************
+    * @description Verifies legacy rollup is used when Customizable Rollups is not activated
+    */
+    @isTest
+    private static void contactMergeShouldUseLegacyRollupWhenCRLPNotEnabled() {
+        RLLP_OppRollupMock mock = setRollupMock();
+
+        List<Contact> contactsToMerge = buildContactsWithOppsForAccount();
+
+        Test.startTest();
+        merge contactsToMerge[0] contactsToMerge[1];
+        Test.stopTest();
+
+        System.assert(mock.rollupContactsCalled, 'Legacy rollup should have been called');
+    }
+
+    /*******************************************************************************************************
+    * @description Verifies legacy rollup is NOT used when Customizable Rollups is activated
+    */
+    @isTest
+    private static void contactMergeShouldUseCRLPWhenEnabled() {
```

Comment:
Since we can't verify that the CRLP is being used, should we rename this to contactMergeShouldNotUseLegacyRollupWhenCRLPEnabled?